stan/math/opencl/matrix_cl.hpp

```diff
@@ -452,15 +452,15 @@ class matrix_cl<T, require_arithmetic_t<T>> {
   }
 
   /**
-   * Construct from a kernel generator expression. It evaluates the ixpression
+   * Construct from a kernel generator expression. It evaluates the expression
    * into \c this.
    * @tparam Expr type of the expression
    * @param expression expression
    */
   template <typename Expr,
             require_all_valid_expressions_and_none_scalar_t<Expr>* = nullptr>
-  matrix_cl(const Expr& expresion);  // NOLINT This constructor is intentionally
-                                     // implicit
+  matrix_cl(const Expr& expression);  // NOLINT This constructor is
```

Comment:
We've just used `NOLINT(runtime/explicit)` I think elsewhere to indicate we're explicitly turning off just that piece of lint.

Reply:
Fixed.